### ISSUE TITLE
Vessl Workload status pending

### ISSUE THREAD

**Issue by crescmoon** [LISTatSNU/FastMRI_challenge#242]
안녕하세요,

Vessl에서 매뉴얼을 따라 first time setup을 진행하고자 했는데, workspace 생성 이후 log에서 다음과 같은 메시지만 반복되며 접속되지 않는 문제가 있어 문의드립니다. Node 번호는 iabeng28입니다.

![image](https://github.com/LISTatSNU/FastMRI_challenge/assets/73477326/135fd310-cb6f-406c-b337-818d357703cb)

**Comment by yxxshin** [comment]
확인 후 Vessl 측에 문의드렸습니다. 답변 받는 대로 알려드리겠습니다.

감사합니다.

**Comment by superjun00** [comment]
저도 정확히 같은 문제가 발생했습니다.  workspace를 지우고 다시 시도해봐도 해결되지 않았습니다. Node 번호는 iabeng55입니다. 같이 문의해 주실 수 있을지 여쭙고 싶습니다.

**Comment by yxxshin** [comment]
확인하였습니다. IABENG55도 같이 문의하겠습니다.

**Comment by yxxshin** [comment]
IABENG28과 IABENG55 이슈가 모두 해결되었습니다. @crescmoon  참가자께서는 튜토리얼 영상대로 새로운 workspace를 하나 더 만드셔서 사용하시기 바랍니다. @superjun00 참가자께서는 기존에 열어두셨던 workspace로 접속하시면 됩니다.

혹시 문제가 발생하면 다시 문의해 주세요.
감사합니다.

**Comment by smtamh** [comment]
오늘 gpu 회수 관련 이메일을 받아서 문의드립니다. 저도 같은 문제가 발생했어서 팀원이 만든 workspace에 토큰을 입력해서 접속했었는데, 혹시 workspace는 팀당 1개만 있으면 gpu 회수 없이 사용할 수 있는건가요? iabeng26 이고, 현재는 workspace 1개만 만들어둔 상태입니다.

**Comment by yxxshin** [comment]
@smtamh 네, 맞습니다! 애초에 workspace를 서버 당 하나씩 밖에 실행하지 못해서, 팀이 두 분이시면 공유하며 사용하셔야 합니다.

**Comment by smtamh** [comment]
네 알겠습니다. 감사합니다!

**Comment by shmin2037** [comment]
@yxxshin 
저도 동일한 문제를 겪고 있는데 혹시 조치 해 주실 수 있나요? IABENG 65

**Comment by yxxshin** [comment]
@shmin2037 안녕하세요,

IABENG 65의 경우 물리적으로 직접 서버 확인을 해 보아야 할 것 같아 시간이 좀 오래 걸릴 것 같습니다 ㅠㅠ
다음 GPU 재할당까지 문제가 해결되지 않는다면, 우선적으로 새 서버로 재할당 해드리겠습니다.

감사합니다.

**Comment by shmin2037** [comment]
@yxxshin 헉 알겠습니다 ㅜㅜ 감사합니다.

**Comment by gates02** [comment]
@yxxshin 
IABENG95를 이번 주에 할당 받아 workspace를 만들어보고 있는데 똑같은 문제가 발생하고 있습니다. 혹시 확인해주실 수 있을까요?

**Comment by yxxshin** [comment]
@gates02 안녕하세요,

위에 언급했듯 서버 당 하나의 workspace 만 실행시킬 수 있으셔서, 팀이 두 분이시면 같은 workspace를 공유하며 사용하셔야 합니다.

감사합니다.

**Comment by gates02** [comment]
@yxxshin 조교님, 답변 감사합니다.
답변 확인하고 원래 있던 workspace를 모두 삭제하고 하나를 다시 만들어봤는데, 그래도 계속 pending 상태가 지속됩니다. 더 기다려봐야 할까요?

**Comment by yxxshin** [comment]
<img width="1266" alt="Screenshot 2024-08-04 at 1 12 45 PM" src="https://github.com/user-attachments/assets/d7587ea0-51df-4a5e-9c55-7f5898f6570b">

안녕하세요, 현재 위와 같이 martian35@snu.ac.kr 님께서 만드신 workspace가 Running 이라고 뜨는데요. 
혹시 같은 팀의 팀원 분이 아니신 것일까요?

감사합니다.

**Comment by Racul** [comment]
@yxxshin 
이번에 iabeng36을 새로 받아 workspace 생성 중에 동일한 문제가 발생하였습니다. other workspace는 없는 것으로 보입니다. 확인 부탁드립니다.

**Comment by yxxshin** [comment]
@racul 안녕하세요,

IABENG 36 해결되었습니다. 감사합니다.
